api/preprints/serializers.py

```diff
@@ -500,6 +502,7 @@ class PreprintCreateSerializer(PreprintSerializer):
     # Overrides PreprintSerializer to make id nullable, adds `create`
     # TODO: add better Docstrings
     id = IDField(source='_id', required=False, allow_null=True)
+    manual_doi = ser.CharField(write_only=True, required=False)
```

Comment:
For myself: Yes! This belongs here in `PreprintCreateSerializer` instead of `PreprintSerializer`. I will update `manual_guid` to be declared here too.